doc/api/deprecations.md

```diff
@@ -95,12 +95,16 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
+### DEP0007: Replace cluster worker.suicide with worker.exitedAfterDisconnect
 
-Type: Runtime
+Type: End-of-Life
```

Comment:
This is a fixed term as defined in [deprecations.md](https://github.com/nodejs/node/blob/master/doc/api/deprecations.md), so I don't think we want to change this.